plane/source/docs/quadplane-parameters.rst

```diff
@@ -29,6 +29,37 @@ Key Parameters
    throttle while flying or you risk disarming your motors.
 -  The default :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>` for a QuadPlane is to 300 (Hz). Most QuadPlanes do not need this to be rasied. Some very small vehicles (< 1Kg) might benefit from setting it to 400. In heavier vehicles, their higher inertia results in lower effective control response rates, so they do not benefit from a higher loop rate. Raising above 300 only leads to larger log files in these vehicles.
 
+.. _return_behavior_setup:
+
+Return Behavior QuickSetup Guide:
+=================================
+
+While there are many parameters setting distances and heights for the various home return modes/behaviors (read each mode's wiki page), this is a quick setup guide for basic behaviors.
+
+1.If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in fixed wing flight set :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` value below to obtain various behavior options:
+- "0" : do nothing if in AUTO mode, otherwise switch to RTL mode
+- "1" : switch to RTL mode
+- "2" : cut throttle and glide in FWBA mode
+- "3" : deploy parachute (assuming you have one setup)
+- "4" : switch to AUTO mode and execute mission at current mission sequence pointer
+
+2. If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in VTOL flight, you will immediately QLAND, unless you select the following optional behavior:
+- set :ref:`Q_OPTIONS<Q_OPTIONS>` bit 5 to switch to QRTL mode instead. (see #5 below)
+- set :ref:`Q_OPTIONS<Q_OPTIONS>` bit 20 to switch to RTL mode instead. (see #3,4 below)
+
+3. Anytime you switch to RTL (due either to manual mode change or failsafe action), do you want to execute an autoland sequence (does not need to actually have a land command, if some other action is desired), rather than just return and loiter around home?
+- If yes,then set up a DO_LAND_START mission sequence and enable the :ref:`RTL_AUTOLAND<RTL_AUTOLAND>` parameter. See :ref:`do_land_start` for details of setup.
```

Comment:
Sure - but when you click the link and get to do_land_start you will find no reference to VTOL/Q modes. I think this is confusing.

Reply:
added note on that page.....the feature itself is VTOL/FW agnostic....the NAV_LAND command evokes the do_land_start tag itself....

Reply:
Ok - this is more of a question, does this auto land behavior apply to RTL only, or would it also apply to QRTL? Either way it might be good to call it out explicitly.

Reply:
its named RTL_AUTOLAND....RTL mode (a fixed wing mode of flight) normally just returns and loiters..but can do the do_land_start sequence instead
and QRTL mode always lands in the end...with or without a fixed wing approach
if you are in VTOL flight switching to RTL mode can do the behaviors as selected in #4...if its selected to be RTL mode, the RTL_AUTOLAND can affect it, since its RTL

Reply:
> its named RTL_AUTOLAND....RTL mode (a fixed wing mode of flight) normally just returns and loiters..but can do the do_land_start sequence instead
> and QRTL mode always lands in the end...with or without a fixed wing approach
> if you are in VTOL flight switching to RTL mode can do the behaviors as selected in #4...if its selected to be RTL mode, the RTL_AUTOLAND can affect it, since its RTL

I'd take exactly those words and put them on the page.